js/script.js

```diff
@@ -0,0 +1,137 @@
+pickScissors.addEventListener('click', function() { playerPick('scissors') });
+
+
+//Wyświetlanie głównych elemntów strony: przycisku, tablicy wyboru i tablicy wyników
+function setGameElements() {
+    switch(gameState) {
+      case 'started':
+          newGameElem.style.display = 'none';
+          pickElem.style.display = 'block';
+          resultsElem.style.display = 'block';
+        break;
+      case 'ended':
+          newGameElem.style.display = 'block';
+          newGameBtn.innerText = 'Jeszcze raz';
+          resultsElem.style.display = 'block';
+          pickElem.style.display = 'none';
+        break;
+      case 'notStarted':
+      default:
+          newGameElem.style.display = 'block';
+          pickElem.style.display = 'none';
+          resultsElem.style.display = 'none';
+    }
+}
+
+setGameElements();
+
+//Liczenie punktów
+function setGamePoints() {
+    playerPointsElem.innerHTML = player.score;
+    computerPointsElem.innerHTML = computer.score;
+}
+
+//Rozpoczęcie gry
+function newGame() {
+    player.name = prompt('Please enter your name', 'imię gracza');
+    if (player.name) {
+      player.score = computer.score = 0;
+
+      playerPickElem.innerText = "Player selection";
+      computerPickElem.innerText = "Computer selection";
+      playerResultElem.innerText = "Player Score";
+      computerResultElem.innerText = "Computer Score";
+
+      pickElem.style.display = 'block';
+      winnerAlertElem.innerText = "";
+
+      gameState = 'started';
+      setGameElements();
+  
+      playerNameElem.innerHTML = player.name;
+      setGamePoints();
+    }
+}
+
+//Wybór gracza
+function playerPick(playerPick) {
+    var computerPick = getComputerPick();
+    
+    playerPickElem.innerHTML = playerPick;
+    computerPickElem.innerHTML = computerPick;
+
+    checkRoundWinner(playerPick, computerPick);
+}
+
+//Wybór komputera
+function getComputerPick() {
+    var possiblePicks = ['rock', 'paper', 'scissors'];
+    return possiblePicks[Math.floor(Math.random()*3)];
+}
+
+//Sprawdzenie kto wygrał rundę
+function checkRoundWinner(playerPick, computerPick) {
+    playerResultElem.innerHTML = computerResultElem.innerHTML = '';
+  
+    if (playerPick == computerPick) {
+        // remis
+    } else if (
+        (computerPick == 'rock' &&  playerPick == 'scissors') ||
+        (computerPick == 'scissors' &&  playerPick == 'paper') ||
+        (computerPick == 'paper' &&  playerPick == 'rock')) {
+  
+        computerResultElem.innerHTML = "Win!";
+        computer.score++;
+    } else {
+        playerResultElem.innerHTML = "Win!";
+        player.score++;
+    }
+
+    setGamePoints();
+    gameOver();
+}
+
+//Zakończenie gry
+function gameOver() {
+    if (player.score === 10 || computer.score === 10) {
+        gameState = 'ended';
+        setGameElements();
+        player.score === 10 ? (winnerAlertElem.innerText = "The winner is: " + player.name) : (winnerAlertElem.innerText = "The winner is: Computer");
```

Comment:
albo tak:
`alertElem.innerText = player.score === 10 ? "The winner is: " + player.name : "The winner is: Computer"`

albo tak:

```
var winner = player.score === 10 ? player.name : "Computer";
alertElem.innerText = "The winner is: " + winner;
```

będzie krócej i czytelniej